pkg/sql/conn_executor_exec.go

```diff
@@ -502,10 +502,8 @@ func (ex *connExecutor) execStmtInOpenState(
 		// Enforce license policies. Throttling can occur if there is no valid
 		// license or if the existing one has expired.
 		if isSQLOkayToThrottle(ast) {
-			if notice, err := ex.server.cfg.LicenseEnforcer.MaybeFailIfThrottled(ctx, curOpen); err != nil {
+			if _, err := ex.server.cfg.LicenseEnforcer.MaybeFailIfThrottled(ctx, curOpen); err != nil {
```

Comment:
it looks like this was the only usage of the `notice` return value. should we update the function signature so it stops returning a notice?

Reply:
I kept it in because it's an effective way to test the logging message. I can either remove it or change the type to a string. Which option do you prefer?